### ISSUE TITLE
[Accessibility Issue] : missing aria label for the profile dropdown

### ISSUE THREAD

**Issue by skarthikeyan96** [EddieHubCommunity/LinkFree#2016]
### What would you like to share?

Aria label missing for the profile dropdown

**Lighthouse audit of the production site**

![Screenshot 2022-10-22 at 2 49 02 PM](https://user-images.githubusercontent.com/23126394/197331691-36c22539-1fd8-4fb7-9cbc-6628f82b16f6.png)
